webui.py

```diff
@@ -1852,6 +1884,7 @@ def run_RealESRGAN(image, model_name: str):
     'Save grid',
     'Sort samples by prompt',
     'Write sample info files',
+    'Write sample info to one file',

```

Comment:
This naming is misleading.
It will make users think that if they tick both individual sample files will not be written.

Reply:
If your goal is a log file: "Write sample log files"?

Reply:
No wait, your files are explicitly not per sample.
"Write log files"?